lib/disk/modules/QcowDisk.rb

```diff
@@ -252,6 +282,14 @@ def read_entries(n)
     entries
   end
 
+  def refcount_order
+    version < 3 ? 4 : @header['refcount_order']
+  end
+
+  def header_length
+    version < 3 ? 72 : @header['header_length']
```

Comment:
Yes and also from the qcow spec:

https://github.com/qemu/QEMU/blob/master/docs/specs/qcow2.txt#L115